legion/sdk/legion/sdk/config.py

```diff
@@ -422,6 +422,15 @@ def __new__(cls, name, default=None, cast_func=str, description=None, configurab
 EDI_TOKEN = ConfigVariableDeclaration('EDI_TOKEN', None, str,
                                       'Token for EDI server authorisation',
                                       True)
+EDI_REFRESH_TOKEN = ConfigVariableDeclaration('EDI_REFRESH_TOKEN', None, str,
+                                              'Refresh token',
+                                              True)
+EDI_ACCESS_TOKEN = ConfigVariableDeclaration('EDI_ACCESS_TOKEN', None, str,
```

Comment:
This config var is never used. Do we need it?

Reply:
In future, I guess, yes. But for now - no, we don't need